marketing.go

```diff
@@ -1,13 +1,23 @@
 package hubspot
 
+import "fmt"
+
 const (
 	marketingBasePath = "marketing"
 )
 
 type Marketing struct {
-	Email MarketingEmailService
+	Email         MarketingEmailService
+	Transactional TransactionalServiceOp
```

Comment:
```suggestion
	Transactional TransactionalService
```

Reply:
Here suppose to have an interface the same as `MarketingEmailService`.

Reply:
This should be fixed.

Reply:
FWIW, I don't think theres a huge case for an interface here as no other object is ever going to reimplement these things.  Test cases could cover the concern that a function gets accidentally renamed or un-exposed.

OTOH, these functions are very difficult to write tests for without a dedicated HubSpot account for that purpose or mocking out their endpoints.

I'm happy to match style, so this is all set and its exposing the interface.

Reply:
Thank you for the update.  The intention to keep this field as interface is let this mockable. 
We can mock in http client layer to prevent external API call, but we also keep the capability to replace implementation by mock in this layer.